apps/code-examples/src/app/code-examples/angular-tree-component/angular-tree/inline-help/angular-tree-demo.component.ts

```diff
@@ -0,0 +1,52 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-angular-tree-component-demo',
+  styles: [
+    `
+      .app-demo-container {
+        border: 1px solid #cdcfd2;
+        padding: 20px;
+
+        .angular-tree-component {
+          background: #fff;
+        }
+      }
+    `,
```

Comment:
Let's throw these styles in a separate `angular-tree-demo.component.scss` file for consistency's sake.